README.md

```diff
@@ -58,12 +58,40 @@ If you also want to run the MySQL tests, ensure that MySQL is installed,
 and then type the following:
 
     mysql -uroot
-    create database hibernaterx;
+    create database hibernate-rx;
     create user "hibernate-rx" identified by 'hibernate-rx';
-    grant all on hibernaterx.* to "hibernate-rx";
+    grant all on hibernate-rx.* to "hibernate-rx";
```

Comment:
@DavideD This isn't right. MySQL can't have a database named `hibernate-rx`, at least not on Mac.

Reply:
Hrrrrm, there might be a way to do it with `mysqladmin` rather than `mysql` but I can't seem to quite get it right.

Reply:
OK, I was wrong. I figured it out.

Reply:
OK, @DavideD, I figured out what I was doing wrong. In the end you just need to quote the database name like [this](https://github.com/hibernate/hibernate-rx/commit/9122e4dba8d7915e8512ba0770a5d576b7e0e233).

But note that the code you have above won't work, at least not on Mac, but I would be surprised if it worked on any platform.